app/jinja_filters.py

```diff
@@ -82,7 +85,15 @@ def format_month_year_date(value, date_format='%B %Y'):
 
 @blueprint.app_template_filter()
 def format_datetime(value, date_format='%d %B %Y at %H:%M'):
-    return "<span class='date'>{date}</span>".format(date=datetime.strptime(value, '%Y-%m-%dT%H:%M:%S.%f').strftime(date_format))
+    date = datetime.strptime(value, '%Y-%m-%dT%H:%M:%S.%f')
+    tz_time = pytz.timezone('UTC').localize(date)
+    london_tz = pytz.timezone('Europe/London')
+    london_time = tz_time.astimezone(london_tz)
+    time = london_time.strftime(date_format)
+    if london_time.tzinfo._utcoffset == timedelta(hours=1):
```

Comment:
The tzinfo object already implements a `dst()` method to work this out.

Reply:
Should we use the `if` to create a variable called something like timezone and then pass that as another format variable. Then we're left with a single return and lessen duplication.
`return "<span class='date'>{date} {timezone}</span>".format(date=datetime.strptime(time, date_format), timezone=timezone)`

Reply:
I also believe the correct format for time zones would be (GMT) or (GMT+1). In other words surround the whole timezone by the bracket not just the +1.